coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,172 @@
+        ".xlf": "XML",
+        ".hh": "C++",
+        ".go": "Go",
+        ".hs": "Haskell",
+        ".phpt": "PHP",
+        ".c++": "C++",
+        ".tex": "TeX",
+        ".rbx": "Ruby",
+        ".irbrc": "Ruby",
+        ".fpp": "FORTRAN",
+        ".rbw": "Ruby",
+        ".php": "PHP",
+        ".tmpreferences": "XML",
+        ".sjs": "JavaScript",
+        ".zcml": "XML",
+        ".sublime-keymap": "JSON",
+        ".html": "HTML",
+        ".vhdl": "VHDL",
+        ".f77": "FORTRAN",
+        ".pac": "JavaScript",
+        ".scxml": "XML",
+        ".gyp": "Python",
+        ".xpy": "Python",
+        ".sublime-workspace": "JSON",
+        ".veo": "Verilog",
+        ".aux": "TeX",
+        ".cproject": "XML",
+        ".sublime_session": "JSON",
+        ".nqp": "Perl",
+        ".yaml": "YAML",
+        ".mkiv": "TeX",
+        ".wsgi": "Python",
+        ".glade": "XML",
+        ".scala": "Scala",
+        ".yml": "YAML",
+        ".mkii": "TeX",
+        ".scss": "SCSS",
+        ".jsfl": "JavaScript",
+        ".sublime-project": "JSON",
+        ".xhtml": "HTML",
+        ".wxi": "XML",
+        ".wxl": "XML",
+        ".rdf": "XML",
+        "._js": "JavaScript",
+        ".for": "FORTRAN",
+        ".cson": "CoffeeScript",
+        ".xsd": "XML",
+        ".tmsnippet": "XML",
+        ".bib": "TeX",
+        ".wxs": "XML",
+        ".h": "C++",
+        ".rst": "reStructuredText",
+        ".m": "Objective-C",
+        ".rss": "XML",
+        ".psc1": "XML",
+        ".psgi": "Perl",
+        ".tpp": "C++",
+        ".c": "C++",
+        ".bones": "JavaScript",
+        ".f": "FORTRAN",
+        ".lua": "Lua",
+        ".mkvi": "TeX",
+        ".vxml": "XML",
+        ".aw": "PHP",
+        ".mxml": "XML",
+        ".py": "Python",
+        ".r": "R",
+        ".v": "Verilog",
+        ".w": "C",
+        ".tmlanguage": "XML",
+        ".f90": "FORTRAN",
+        ".f95": "FORTRAN",
+        ".plist": "XML",
+        ".ts": "TypeScript",
+        ".x3d": "XML",
+        ".md": "Markdown",
+        ".xml": "XML",
+        ".toc": "TeX",
+        ".tcc": "C++",
+        ".mm": "Objective-C",
+        ".tmtheme": "XML",
+        ".clixml": "XML",
+        ".podspec": "Ruby",
+        ".pod": "Perl",
+        ".cmake": "CMake",
+        ".xaml": "XML",
+        ".dart": "Dart",
+        ".xmi": "XML",
+        ".jelly": "XML",
+        ".svg": "XML",
+        ".coffee": "CoffeeScript",
+        ".axml": "XML",
+        ".nse": "Lua",
+        ".ron": "Markdown",
+        ".kml": "XML",
+        ".jsx": "JavaScript",
+        ".cuh": "Cuda",
+        ".jss": "JavaScript",
+        ".jsm": "JavaScript",
+        ".pluginspec": "XML",
+        ".cmake.in": "CMake",
+        ".gemspec": "Ruby",
+        ".god": "Ruby",
+        ".js": "JavaScript",
+        ".reek": "YAML",
+        ".ins": "TeX",
+        ".ditaval": "XML",
+        ".pyt": "Python",
+        ".pyw": "Python",
+        ".php3": "PHP",
+        ".java": "Java"}
```

Comment:
not an issue, but dont you feel like ordering them by value? It would be way easier to check if any are missing :)